templates/certified/search-results.html

```diff
@@ -202,6 +203,10 @@ <h3 class="p-heading--4">No results - why not try widening your search?</h3>
 
 <script>
 
+ function clearFilters() {
+  location.assign(`${location.protocol}//${location.host}/certified?q=`)
```

Comment:
Do you really need `location.protocol and location.host?` does it not work with `/`?

Reply:
it works with `/certified?q=` - updated, thanks!